tools/plotting.py

```diff
@@ -221,6 +221,9 @@ def make_data_mc_comparison_plot( histograms = [],
     axes=None
     if show_ratio:
         ratio = data.Clone( 'ratio' )
+        sumHists = sum( stack.GetHists() )
+        for bin in range(1,sumHists.GetNbinsX()):
```

Comment:
I know we have this in a few places in the code already, but it would be good if you could rename `bin` to `bin_i`.
`bin` is a python built-in function: https://docs.python.org/2/library/functions.html#bin